### PR TITLE
[MAID-3107] chore/ci: update scripts to Rust stable 1.28.0

### PR DESCRIPTION
**Pull request by octol**
nightly to 2018-07-07
clippy to 0.0.212
rustfmt to 0.99.2

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,8 +19,8 @@ before_script:
   - curl -sSL https://github.com/maidsafe/QA/raw/master/travis/cargo_install.sh > cargo_install.sh
   - bash cargo_install.sh cargo-prune;
   - if [[ $TRAVIS_RUST_VERSION =~ ^nightly.* && $TRAVIS_OS_NAME == linux ]]; then
-      bash cargo_install.sh rustfmt-nightly 0.8.2;
```
Comment (mrcnski) [inline .travis.yml:22]: Can we move these version strings into variables like in our other CI scripts? The rest looks great 👍

**Review by mrcnski — DISMISSED** [review]

File: .travis.yml (hunk 1 above)
Comment (octol) [reply .travis.yml:22]: Will do!

File: .travis.yml [hunk 2]
```diff
@@ -2,13 +2,17 @@ env:
   global:
     - RUST_BACKTRACE=1
     - PATH=$PATH:$HOME/.cargo/bin
+    - RUST_STABLE=1.28.0
```
Comment (pierrechevalier83) [inline .travis.yml]: I would remove these 2 lines as these variables are unused

**Review by pierrechevalier83 — CHANGES_REQUESTED** [review]
I would stick to environment variables that are needed

**Review by pierrechevalier83 — APPROVED** [review]

**Review by nbaksalyar — APPROVED** [review]

**Pull request merged**